livre/eq/Channel.cpp

```diff
@@ -376,8 +332,8 @@ struct Channel::Impl
         glMatrixMode( GL_MODELVIEW );
```

Comment:
What's that !? It's deprecated for a long time and we are explicitly giving all the matrices to our shaders. 
Why is this nedeed ?